newm-server/src/main/kotlin/io/newm/server/database/migration/V59__MarketplaceUpdates.kt

```diff
@@ -0,0 +1,29 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V59__MarketplaceUpdates : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
```

Comment:
@wlara Please add default values for all the new config keys here.

Reply:
Done